pypeclub/OpenPype
# Hiero: loading effect family to timeline

## Brief description
Loader for loading effects to timeline

## Description
Hiero is having available loader for multi loading of effects to active timeline

## Additional info
- also fix of multiselected loading

## Testing notes:
1. open vfx testing project
2. open loader at any active timeline
3. select effect subset in multiselected assets
4. chose load effect and see all effects were loaded
